### ISSUE TITLE
[Feature Request]

### ISSUE THREAD

**Issue by PsychoAnees** [JunkFood02/Seal#383]
### Checklist

- [ ] This feature I'm requesting is already implemented in yt-dlp.
- [ ] This feature is merely a UI/UX update.
- [x] This feature is suitable for primary users with little knowledge about yt-dlp.
- [X] This feature is available for most websites, not only the video platform I use.
- [X] This feature is suitable for a large variety of videos.
- [X] This feature is not going to conflict with many of the existing options.

### Is your feature request related to a problem? Please describe.

its not related to a problem but if it is implemented then it would be the best Audio downloader too.
im basically asking for multiple bitrate options when downloading audio for a youtube video or any other video.

### Describe the solution you'd like

I want multiple kbps option when download an audio.
like when i put a link of youtube video we only get a default option to download audio file of that video,but i want multiple bit rates option for better audio quality.
basically it will become a music downloader too.
i will add a picture of a website that is used to download mp3 music
so it will be easy for you to understand
![Screenshot_20221203_205722_Opera](https://user-images.githubusercontent.com/96819416/205449990-9fa0d39f-0a05-48c5-9ed9-1766b62d817c.png)

### Additional context

![Screenshot_20221203_205722_Opera](https://user-images.githubusercontent.com/96819416/205450006-b4fe8ed3-1e97-433f-9551-07937c8add3a.png)

**Comment by JunkFood02** [comment]
See also #376 #118 #249

**Comment by PsychoAnees** [comment]
> See also #376 #118 #249 #228

theb can you plz just tell me a custom command to download songs from youtube at 320kbps

**Comment by Marco-9456** [comment]
> 

This is impossible without a Youtube Premium subscription, if you have one then you can take advantage of it and use the `--cookies` command

**Comment by Marco-9456** [comment]
> > See also #376 #118 #249 #228
> 
> theb can you plz just tell me a custom command to download songs from youtube at 320kbps

This is impossible without a Youtube Premium subscription, if you have one then you can take advantage of it and use the `--cookies` command

**Comment by PsychoAnees** [comment]
> 

i have revanced youtube ,all like premium.
so will it work?
if yes then can you plz describe the steps as im noob

**Comment by Marco-9456** [comment]
> > 
> 
> i have revanced youtube ,all like premium. so will it work? if yes then can you plz describe the steps as im noob

1. No it will not work, because the account itself isn't Premium.
2. According to [Google](https://support.google.com/youtubemusic/answer/9076559?hl=en) the highest bitrate they support is `256kbps` .

**Comment by PsychoAnees** [comment]
so you mean i can download 256kbps only.
so how can i download audio at this specific bitrate

**Comment by Marco-9456** [comment]
> so you mean i can download 256kbps only. so how can i download audio at this specific bitrate

1. You ***NEED*** Youtube Premium Subscription to be able to do this 
2. Add this [Extension](https://chrome.google.com/webstore/detail/get-cookiestxt/bgaddhkoddajcdgocldbbfleckgcbcid)  to a browser called [Kiwi](https://play.google.com/store/apps/details?id=com.kiwibrowser.browser) 
3. Login to Youtube Music in Kiwi
4. Open the 3 dots menu and click on Get cookies.txt extension
5. Click on export and a .txt file will be downloaded
6. Open it and copy everything
7. Now go to Seal -> Settings -> Network -> Cookies then paste what you have copied and then go back and toggle the Cookies option on.
8. And now you're done every time you want to download something from Youtube or youtube Musik it will be download in the best available quality.
